pro/src/components/StockEventForm/constants.ts

```diff
@@ -11,3 +11,13 @@ export const STOCK_EVENT_FORM_DEFAULT_VALUES: IStockEventFormValues = {
   price: '',
   isDeletable: true,
 }
+
+// 'price','quantity','bookingLimitDatetime', are editable
+export const STOCK_EVENT_ALLOCINE_NOT_EDITABLE_FIELDS = [
+  'stockId',
+  'beginningDate',
+  'beginningTime',
+  'remainingQuantity',
+  'bookingsQuantity',
+  'isDeletable',
```

Comment:
```suggestion
  'beginningDate',
  'beginningTime',
```

Les autres champs sont des champs caché, non éditable quoi qu'il arrive.